roles/kubernetes/control-plane/defaults/main/main.yml

```diff
@@ -118,6 +118,7 @@ kube_webhook_token_auth_url_skip_tls_verify: false
 kube_webhook_authorization: false
 ## Variables for webhook token authz https://kubernetes.io/docs/reference/access-authn-authz/webhook/
 # kube_webhook_authorization_url: https://...
+# kube_webhook_token_auth_ca_data:
```

Comment:
It would be good to add this to the sample inventory file and add some description + sample formatted data.

Reply:
I can add this. Would you recommend adding it to sample/group_vars/all/all.yml?

Reply:
That seems like a good place, yes